supabase/migrations/20240806124555_task_sales_id.sql

```diff
@@ -0,0 +1 @@
+alter table "public"."tasks" add column "sales_id" bigint;
```

Comment:
**issue:** You forgot to update the `Task` type in `types.ts` to reflect the change